app/driveshare.html

```diff
@@ -308,6 +258,8 @@ <h4 class="modal-title">Console Logs</h4>
 	.script("lib/electron_boilerplate/env_config.js")
 	.script("lib/electron_boilerplate/context_menu.js")
 	.script("lib/electron_boilerplate/external_links.js")
+	/*.script('bower_components/angular/angular.js')
```

Comment:
Remove commented block of code unless it's part of a change that is about to happen.